medcat/utils/config_utils.py

```diff
@@ -15,9 +15,24 @@ def weighted_average_function(self) -> Callable[[float], int]:
 
 
 def is_old_type_config_dict(d: dict) -> bool:
```

Comment:
This assumes all configs were created post the pydantic changes to the config right?

Reply:
Yes, that would be true since before that it would have been saved as a raw dict.